std_msgs/Header header

# following reference to std_msg/std_srv
string x_label
uint32 x_size
uint32 x_stride

string y_label
uint32 y_size
uint32 y_stride

float32[] data
